dev/com.ibm.ws.security.mp.jwt/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -50,6 +50,7 @@
             <Option label="%tokenHeader.Cookie" value="Cookie" />
         </AD>
         <AD id="cookieName" name="%cookieName" description="%cookieName.desc" required="false" type="String" ibm:beta="true" />
+        <AD id="keyManagementKeyAlias" name="internal" description="internal use only" required="false" type="String" ibm:beta="true" />
```

Comment:
TODO: Update to use keyManagementKeyAlias and keyManagementKeyAlias.consumer.desc